app/controllers/api/v1/users_controller.rb

```diff
@@ -77,7 +77,7 @@ def user_params
           :company_role,
           :company_name,
           :education_level,
-          :interests
+          :interests => []
```

Comment:
This is supposed to be an array, is that right?
`interests: params[:interest_id]`